src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/Configurations/NLogLogger.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NLog;
+using NLog.Extensions.Logging;
+using NLog.Web;
+using System;
+using LogLevel = Microsoft.Extensions.Logging.LogLevel;
+
+namespace Exadel.CrazyPrice.Common.Configurations
+{
+    public class NLogLogger : ICrazyPriceLogger
+    {
+        private Logger _logger;
+        private readonly IHostBuilder _hostBuilder;
+        private readonly IConfigurationRoot _configuration;
+
+        public NLogLogger(IHostBuilder hostBuilder, IConfigurationRoot configuration)
+        {
+            _hostBuilder = hostBuilder;
+            _configuration = configuration;
+        }
+
+        public IHostBuilder UseLogger()
+        {
+            LogManager.Configuration = new NLogLoggingConfiguration(_configuration.GetSection("NLog"));
```

Comment:
Configuration is specific for each logger - remove this logic to upper level

Reply:
Removed.